extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -547,10 +546,10 @@ export class AmpConsent extends AMP.BaseElement {
         if (!this.uiInit_) {
           this.uiInit_ = true;
           toggle(this.element, true);
-          this.getViewport().addToFixedLayer(this.element);
         }
         this.element.classList.add('amp-active');
         this.element.classList.remove('amp-hidden');
```

Comment:
I noticed `this.element.classList` was requested four times during the invocation of this function. Is there a reason it cannot be stored and referenced? From a quick glance, this would reduce around 53 bytes from the pre-gzipped, but minified size of the extension. 

Example:
```
handlePostPromptUI_() {
    const classList = this.element.classList;
    ...
    this.vsync_.mutate(_ => {
      classList.add('amp-active');
      classList.remove('amp-hidden');
      ...
    });
});

Reply:
good point! fixed. 
By storing the `classList`, what exactly can be save? What will the `this.element.classList` be minimized to?